maxdrapa.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "@maxdrapa"
+project: "skiLight"
+---
+
+# skiLight
+
+## Summary
+
+I am planning on integrating a custom lighting system into my ski helmet to act as headlights. I hope to be able to control this wirelessly through Bluetooth and even add some fun rainbow lighting. As a Vermonter, I'm always outside in the winter, and being able to have lights that won't take up any more space on my helmet is optimal. With this, my nighttime skiing and sledding will be improved greatly.
+
+## Plan
+
+This will be an ambiguous project due to the fact that hacking something to look like it was built that way is quite difficult. :/ Here’s my plan though:
+
+1: Take measurements of EVERYTHING so I am able to CAD up a design of my project. (This will tell me if it's possible in the form factor I’m going for)
+
+2: 3D print and purchase all the parts I will be using, assuming the plans won’t change.
+
+3: Shove all the components into my custom case that I printed and wire the heck out of it. I will make sure to test all parts and only run power through it when I know it's safe.
+
+4: Using the few programming skills I have, I will attempt to make a simple program that turns on and off the lights with the push of a button. After I know the circuit is working, I can move on to the next step... the actual coding.
+
+5: With a little bit more knowledge of coding I will attempt to make a little app that allows me to connect to and control the lights via Bluetooth.
+
+6: Now comes the hard part, making everything fit and finished. I will use every sealant imaginable to adhere my pieces to a cutout cavity in the helmet. (In the process I hope not to ruin any internal structures) Then I will cover up every little detail until the helmet looks fresh off the assembly line. 
+
+7: Finally, the fun part, actually using it. Of course, I will have to test it out and actually use it because then there would be no point in making it. This will also help me find any weaknesses that must be fixed
+
+8: Completely optional based on time, but I might add some RGB lights into the sides and top of my helmet that will add some fun effects. (These too will be connected to the microcontroller.) 
+
+## Budget
+
+| Product            | Supplier/Link                                        		 	| Cost   |
+| ------------------ | ----------------------------------------------------------------	| ------ |
+| Solder/wires       | Already own                    			        			| $00.00 |
+| Light modules x2   | www.lumensfactory.com/en/product_detail.php?pid=387         	| $77.00 |
+| Li ION battery x2  | www.adafruit.com/product/328				          		| $29.90 |
+| Microcontroller    | www.adafruit.com/product/5400				          		| $19.95 |
+| Helmet             | Already own								          	| $00.00 |
+| Fasteners,glue,etc | Hardware store						          			| $35.00 |
+| Plastic,paint,sandpaper(fit and finishing)| Hardware store							          		| $50.00 |
```

Comment:
Do you think you'll also need passive components, like resistors? Or perhaps a breadboard for prototyping your circuit?

Reply:
I have a large supply of components like that, so I figured I wouldn't neet to put those in. If you would like, I can gladly add them

Reply:
All good! Just wanted to check you had them.